Mlem/Views/Tabs/Inbox/Feed/AllItemsFeedView.swift

```diff
@@ -39,21 +40,31 @@ extension InboxView {
     // delete it, recompile, paste it, and it should work. Go figure.
     @ViewBuilder
     func inboxListView() -> some View {
-        ForEach(allItems) { item in
+        ForEach(inboxTracker.items, id: \.uid) { item in
             VStack(spacing: 0) {
-                Group {
-                    switch item.type {
-                    case let .mention(mention):
-                        inboxMentionViewWithInteraction(mention: mention)
-                    case let .message(message):
-                        inboxMessageViewWithInteraction(message: message)
-                    case let .reply(reply):
-                        inboxReplyViewWithInteraction(reply: reply)
-                    }
-                }
+                inboxItemView(item: item)
                 
                 Divider()
             }
         }
+        
+        EndOfFeedView(loadingState: inboxTracker.loadingState, viewType: .cartoon)
```

Comment:
I see this on my `test-mlem` account which only has one notification, but not on my main account that has many pages of notifications

Reply:
Hmm, that's odd--it works properly on my main, which also has many pages of inbox, and I tested it with an aggressively reduced page size to trigger loading more often. I can't seem to repro.

Reply:
Ah, you're right, it's actually a different bug. I had thought I'd reached the end but I hadn't - when you scroll down too fast, it doesn't always trigger loading the next page. The easiest way to repro this is by enabling the scroll indicator and dragging the scroll bar all the way to the bottom. The EndOfFeedView loads properly for me when I scroll slowly enough.

Reply:
Fixed--I've added a fallback threshold that triggers on the last item in feed, which seems to work nicely.